src/utils/version.js

```diff
@@ -26,27 +26,26 @@ const version = {
    * @memberof Version
    * @inner
    */
-  checkApiVersion() {
-    const newPromise = new Promise((resolve, reject) => {
-      const libPromise = hathorLib.version.checkApiVersion();
-      libPromise.then((data) => {
-        // Update version allowed in redux
-        store.dispatch(isVersionAllowedUpdate({allowed: hathorLib.helpers.isVersionAllowed(data.version, hathorLib.constants.MIN_API_VERSION)}));
-        // Set network in lib to use the correct address byte
-        let network;
-        if (data.network === 'mainnet') {
-          network = 'mainnet';
-        } else {
-          // Can we assume it will be testnet? I think it's safe
-          network = 'testnet';
-        }
-        helpers.updateNetwork(network);
-        resolve({...data, network});
-      }, (error) => {
-        reject();
-      });
-    });
-    return newPromise;
+  async checkApiVersion(wallet) {
+    const data = await wallet.getVersionData();
+
+    // Update version allowed in redux
+    store.dispatch(isVersionAllowedUpdate({
+      allowed: hathorLib.helpers.isVersionAllowed(data.version, hathorLib.constants.MIN_API_VERSION)
+    }));
+    // Set network in lib to use the correct address byte
+    let network;
+    if (data.network === 'mainnet') {
+      network = 'mainnet';
+    } else {
+      // Can we assume it will be testnet? I think it's safe
+      network = 'testnet';
```

Comment:
On `screens/Server.js` we are already preparing for networks other than `mainnet` and `testnet`.
Should we make any changes here as well? Or at least indicate how to refactor this on a future change?

Reply:
I think we can accept `privatenet` here as well

Refactored in f95c758d204a7d5d9b78befad21794f25b2b52f7

What do you think? @tuliomir and @pedroferreira1

Reply:
For me it seems correct now: both files have the same rules for network naming.